content/en/synthetics/testing_tunnel.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Synthetic Testing Tunnnel
+kind: documentation
+description: Local and CI/CD testing with Datadog Synthetic testing tunnel.
+further_reading:
+- link: "https://www.datadoghq.com/blog/datadog-synthetic-ci-cd-testing/"
+  tag: "Blog"
+  text: "Incorporate Datadog Synthetic tests into your CI/CD pipeline"
+- link: "https://www.datadoghq.com/blog/internal-application-testing-with-datadog/"
+  tag: "Blog"
+  text: "Test internal applications with Datadog's testing tunnel and private locations"
+- link: "/synthetics/browser_tests/"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+- link: "/synthetics/api_tests/"
+  tag: "Documentation"
+  text: "Configure an API Test"
+
+---
+
+<div class="alert alert-warning">
+This feature is currently in public beta.
+</div>
+
+The Synthetic testing tunnel creates **short lived secure connections between your internal environments and the Datadog infrastructure**, allowing you to **swiftly trigger Synthetic tests on your private applications**.  
+Datadog recommends using the testing tunnel if you are willing to launch Synthetics tests from your **CI/CD pipeline** or against **local** versions of your application without deploying a dedicated and long lasting probing system (such as [private locations][1]). The testing tunnel can also prove helpful if you are looking at triggering tests on **ephemeral cloud environments**.
+
+## What is the testing tunnel?
+
+The testing tunnel is a functionality that comes with the **[@datadog/datadog-ci][2] NPM package** which is one of the methods provided by Datadog to include your Synthetic tests as part of your CI/CD pipelines. The testing tunnel creates an **end-to-end encrypted HTTP proxy between your infrastructure and Datadog**, meaning that any test requests sent via the CLI are automatically routed through the `datadog-ci` client, allowing Datadog to access and test your internal applications. 
+
+{{< img src="synthetics/tunnel_diagram.png" alt="Synthetic testing tunnel diagram"  style="width:100%;">}}
+
+`datadog-ci` first gets a presigned URL from Datadog (for authentication). It then opens a secure WebSocket Secure connection (wss) to Datadog's managed locations using the presigned URL. Using SSH connections through the websocket connection, tests are triggered by `datadog-ci` and executed through Datadog's managed locations. Because DNS resolution is performed through the tunnel, it means you can test applications with internal domains or even on the `localhost` of the machine running `datadog-ci`.
```

Comment:
```suggestion
`datadog-ci` first gets a presigned URL from Datadog (for authentication). It then opens a secure WebSocket Secure connection (wss) to Datadog's managed locations using the presigned URL. Using SSH connections through the websocket connection, tests are triggered by `datadog-ci` and executed through Datadog's managed locations.  
Because DNS resolution is performed through the tunnel, it means you can test applications with internal domains or even on the `localhost` of the machine running `datadog-ci`.
```